ci/src/updater.py

```
@@ -30,19 +33,30 @@ def batch_github_plugin_info(info: P, tags: ETagsType) -> P:
     if assets:
         info[url_download] = assets[0]["browser_download_url"]
         info[version] = clean(latest_rel["tag_name"], "v")
+
+    if version_tuple(info[version]) < version_tuple(latest_rel["tag_name"]):
+        tqdm.write(f"Update detected: {info[id_name]} {info[version]}")
+        try:
+            update_hook(webhook_url, info)
+        except:
+            # We dont want to crash if the webhook is not working
```

do we not want it to error rather than fail silently?

---

I was thinking of adding output to console but I dont think it's that important. What we really dont want to happen is have the whole script fail because of a webhook.

---

how do we know if the webhook is working or not though, plugin updates are not often, maybe write a console output before passing?

---

I'll add a `bool` and check at the end of loop to print an error message.

---

why not just write the output above the pass?

---

Prints message out on error